src/tests/ftest/object/same_key_different_value.py

```diff
@@ -87,47 +74,40 @@ def test_single_to_array_value(self):
                 if i == 0:
                     try:
                         # write array value data to same keys, expected to fail
-                        self.container.write_an_array_value(array_value_data,
-                                                            dkey, akey, obj,
-                                                            obj_cls=1)
+                        self.container.write_an_array_value(
```

Comment:
(pylint-no-member) Instance of 'list' has no 'write_an_array_value' member
(pylint-no-member) Instance of 'TestContainer' has no 'write_an_array_value' member